srcpkgs/grub-btrfs/files/grub-btrfs/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+[ -r conf ] && . ./conf
+exec 2>&1
+exec wendy -v -m 768 -w "${SNAPSHOTS_PATH}" bash -c \
```

Comment:
1. Omit the `-v` here and add some logic that will recognize a variable like `VERBOSE` that can be set in the `conf` file to govern the addition of `-v`. This could be a catch-all `OPTS` variable like other services uses, but there probably isn't much value in allowing other flags, since you want tighter control on the path and mode anyway.
2. Is mode 768 sufficient? If snapshots can be renamed, you might also need bits for `IN_MOVE` or the `IN_MOVED_{TO,FROM}`.  You should do some experiments to see what kind of events a snapshot create/rename/destroy trigger.
3. You should probably verify that `SNAPSHOTS_PATH` is non-empty and either fail or substitute a default when it is.